server/controllers/authController.ts

```diff
@@ -56,3 +60,71 @@ export const signIn = async (req: Request, res: Response) => {
     res.status(404).json({ message: error.message });
   }
 };
+
+// *** NEW AUTH ***
+
+export const register = async (req, res) => {
+  const { name, email, password } = req.body;
+
+  try {
+    const hashedPassword = await hash(password, 10);
+    await prisma.user.create({
+      data: { name, email, password: hashedPassword },
+    });
+
+    return res.status(201).json({
+      success: true,
+      message: "The registration was successful",
+    });
+  } catch (error) {
+    console.log(error);
+    return res.status(500).json({
+      error: error.message,
+    });
+  }
+};
+
+export const login = async (req, res) => {
```

Comment:
-not validating user with credentials that are in the db
-add validation before signing the token

Reply:
The user is validated in the backend when doing the login request backend